sources/packages/api/src/services/application/application.service.ts

```diff
@@ -1221,4 +1221,48 @@ export class ApplicationService extends RecordDataModelService<Application> {
       })
       .getMany();
   }
+  /**
+   * Gets active application.
```

Comment:
Please mention something about COE in the completed state. It seems to me that "active application" is too generic.